test-cases/longevity/longevity-alternator-3h.yaml

```diff
@@ -1,22 +1,27 @@
 test_duration: 180
 prepare_write_cmd: [
-  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=20971520 -p fieldcount=10 -p fieldlength=512", ]
+  "bin/ycsb load dynamodb -P workloads/workloadc -threads 80 -p recordcount=20971520 -p fieldcount=10 -p fieldlength=512 -p dataintegrity=true", ]
```

Comment:
@roydahan 
![image](https://user-images.githubusercontent.com/340979/74458663-b493d380-4e92-11ea-9c59-56e5f5483005.png)

what is the avg. CPU we expect on a basic sanity, we should aim for ~50% ? (don't remember why I've lower it, I was trying to understand why the dataintegrity was failing)

I'll put it back up a notch